src/org/jitsi/impl/neomedia/transform/csrc/SsrcTransformEngine.java

```diff
@@ -261,6 +266,17 @@ public RawPacket reverseTransform(RawPacket pkt)
                 csrcAudioLevelDispatcher.addLevels(levels, pkt.getTimestamp());
             }
         }
+
+        SequenceNumberRewriter rewriter = ssrcToRewriter.get(pkt.getSSRCAsLong());
```

Comment:
Not sure I understand this. This is in the audio chain, so packets wouldn't have been dropped elsewhere, so all packets will reach the rewriter. Am I missing something?